test/minikube/minikube_domain_resync_test.go

```diff
@@ -465,4 +488,61 @@ func TestLoadBalancerConfigurationResync(t *testing.T) {
 	options.SetValues["admin.lbConfig.default"] = "random(first(label(node node1) any))"
 	options.SetValues["admin.lbConfig.policies.manual"] = "random(any)"
 	verifyLoadBalancer(t, namespaceName, admin0, options.SetValues)
-}
\ No newline at end of file
+}
+
+func TestKaaLimitedPermissions(t *testing.T) {
+	if os.Getenv("NUODB_LICENSE") != "ENTERPRISE" {
+		t.Skip("This test requires NuoDB 4.1.1+")
+	}
```

Comment:
hmm. This works, but I don't like using the license as a version guard. 

My idea was to create a new file and put in the `enterprise` build guard. Once we upgrade to say 4.1.1 as the default NuoDB instance, we can then move the whole file to say `long`.

Reply:
That works for me. I will move the test to a new file and tag it as `enterprise`.